src/WebJobs.Script.WebHost/WebJobsServiceCollectionExtensions.cs

```diff
@@ -62,6 +63,7 @@ public static IServiceProvider AddWebJobsScriptHost(this IServiceCollection serv
         {
             services.AddWebJobsScriptHostRouting();
             services.AddMvc()
+                .AddJsonOptions(options => options.SerializerSettings.ContractResolver = new DefaultContractResolver())
```

Comment:
Mvc camel cases JSON. This restores the default. One of the tests I'm enabling was verifying casing.